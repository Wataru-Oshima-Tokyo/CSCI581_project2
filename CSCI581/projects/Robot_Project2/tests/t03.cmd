 inp3.txt 0.1 obs3.txt
