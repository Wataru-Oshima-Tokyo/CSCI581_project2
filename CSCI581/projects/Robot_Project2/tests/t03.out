0.6395360171556682 0 
0.9948219099644799 6 
0.9998288465473362 9 
0.9876500014178371 13 
0.3359284930364545 9 
0.4070679039172291 21 
